### ISSUE TITLE
[i18n] - French translation - fr-FR

### ISSUE THREAD

**Issue by AdrienFromToulouse** [philsturgeon/awesome-earth#146]
Translate content to french.

What content should be prioritised?

**Comment by philsturgeon** [comment]
Should we tie localization to the country flag? 

See countries with multiple languages. I don't want to confuse Spain and Spanish. I don't want Spanish speakers in Colombia stuck reading about Spanish projects. 

That said, we dont have any Colombian projects or Spanish translations yet so maybe this is a premature concern? 

Otherwise, we need a Language selector and a Country selector. The country might be able to pick a logical default language for editorial translations, but the country selector focuses on where the projects are available. This would let Korean speakers living in England can see about English projects with descriptions written in Korean. 

Of course only if those translations exist, I am ok defaulting to English or "the reasonable default for a country" if the selected translation does not exist. 

Does this all make sense?

**Comment by AdrienFromToulouse** [comment]
HI, @philsturgeon yep it makes total sense and sorry for late reply. I used to work on similar internationalisation concerns.

May I go a little bit side track? I was thinking that Durpal 8 has now internationalization ready at the core. No need to take care of it like in previous versions.
Would not be "simpler" to refactor with a Drupal and focus on the content?

Question is also: what would be the plan to host it with its DB. There are many "cheap" options, but not as cheap as a static website. 

Happy to hear your thoughts about it.

_My schedule is a bit crowded at the moment but I could setup a Drupal 8 in the following 3 weeks if you wish to have a look_

Cheers,

**Comment by Squallac** [comment]
> Should we tie localization to the country flag?
> 
> See countries with multiple languages. I don't want to confuse Spain and Spanish. I don't want Spanish speakers in Colombia stuck reading about Spanish projects.
> 
> That said, we dont have any Colombian projects or Spanish translations yet so maybe this is a premature concern?
> 
> Otherwise, we need a Language selector and a Country selector. The country might be able to pick a logical default language for editorial translations, but the country selector focuses on where the projects are available. This would let Korean speakers living in England can see about English projects with descriptions written in Korean.
> 
> Of course only if those translations exist, I am ok defaulting to English or "the reasonable default for a country" if the selected translation does not exist.
> 
> Does this all make sense?

I agree with the flags concerns, I wouldn't like the content to be tied with the country when selecting a language. Probably in the future would be cool to have a country filter, to be able to see only the filters in your country. 
But for translation purposes, I'd rather have only the options in text without a flag.

English / Spanish / French / Korean

BTW I'm a native spanish speaker person, I could help with the translation to spanish.
